examples/only-logs-otel/rendered_manifests/configmap-agent.yaml

```diff
@@ -227,6 +227,42 @@ data:
         - from: attributes["log.file.path"]
           to: resource["com.splunk.source"]
           type: move
+        - default: clean-up-log-record
+          routes:
+          - expr: (resource["k8s.namespace.name"]) == "default" && (resource["k8s.pod.name"])
+              matches "buttercup-app-.*" && (resource["k8s.container.name"]) == "server"
+            output: default_buttercup-app-.*_server
+          - expr: (resource["k8s.namespace.name"]) == "k8s-nm2" && (resource["k8s.pod.name"])
+              matches "foo-app-.*" && (resource["k8s.container.name"]) == "client"
+            output: k8s-nm2_foo-app-.*_client
+          - expr: (resource["k8s.namespace.name"]) == "k8s-nm3" && (resource["k8s.pod.name"])
+              matches "foo-app-.*" && (resource["k8s.container.name"]) == "server"
+            output: k8s-nm3_foo-app-.*_server
+          type: router
+        - combine_field: attributes.log
+          combine_with: "\t"
+          id: default_buttercup-app-.*_server
+          is_first_entry: (attributes.log) matches "^[^\\s].*"
+          max_log_size: 1048576
+          output: clean-up-log-record
+          source_identifier: resource["com.splunk.source"]
+          type: recombine
+        - combine_field: attributes.log
+          combine_with: ""
+          id: k8s-nm2_foo-app-.*_client
+          is_first_entry: (attributes.log) matches "^[^\\s].*"
+          max_log_size: 1048576
+          output: clean-up-log-record
+          source_identifier: resource["com.splunk.source"]
+          type: recombine
+        - combine_field: attributes.log
+          combine_with: \n
```

Comment:
Should this be`"\n"` like the tab char? The[ doc mentions](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/recombine.md#configuration-fields) to give input as double quoted instead of literals.

Reply:
Maybe I'll just omit combine_with for the default case since it's assumed to be "\n".